randovania/games/prime1/exporter/scan_tips.py

```diff
@@ -0,0 +1,696 @@
+        # Phendrana Canyon L-Jump Tip
+        # Replaces Phendrana Canyon platform scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;The item in this room can be acquired without the Scan Visor "
+            "by using the &main-color=#FF3333;Space Jump Boots&main-color=#89D6FF; and L-Jumping."
+        ),
+    ],
+    "3740523398": [
+        # Phendrana Canyon Escape Tip
+        # Replaces Phendrana Canyon half-pipe fish scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;As long as the boxes in this room remain intact, "
+            "they can be climbed to exit this room without any items."
+        ),
+    ],
+    "442524052": [
+        # Rapid Fire Missiles Tip
+        # Replaces Main Plaza holobanner
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Hold&image=5D9F9796;and rapidly tap "
+            "&image=EA2A1C5C; to fire missiles as fast as possible."
+        ),
+    ],
+    "807982837": [
+        # Underwater Slope Jump Tip
+        # Replaces Watery Hall plant flavor text
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;Underwater slope jumps performed without the Gravity Suit allow access "
+            "to great heights.\n\nYou can use an underwater slope jump to reach the sunken item in this "
+            "room without the Gravity Suit. This is significantly easier with the "
+            "&main-color=#FF3333;Space Jump Boots&main-color=#89D6FF;."
+        ),
+    ],
+    "2077227745": [
+        # Meta Ridley Boost Tip
+        # Replaces Meta Ridley scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "Meta Ridley",
+        (
+            "&main-color=#89D6FF;During the first phase of combat against Meta Ridley, "
+            "hitting it with a &main-color=#FF3333;Boost Ball&main-color=#89D6FF; "
+            "charge causes enough damage to immediately move to the fight's second phase."
+        ),
+    ],
+    "2967215000": [
+        # Non-Superheated Back Magmoor Tip
+        # Replaces Puddle Spore scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "Puddle Spore",
+        (
+            "&main-color=#89D6FF;Everything south of Twin Fires Tunnel in Magmoor Caverns is not superheated,\n\n"
+            "allowing for safe travel without the Varia Suit."
+        ),
+    ],
+    "1735564172": [
+        # Arboretum Sandstone PBs Tip
+        # Replaces Arboretum sandstone scan
+        "&main-color=#FF3333;Hidden message discovered.",
+        "",
+        (
+            "&main-color=#89D6FF;The &main-color=#FF3333;Sandstone&main-color=#89D6FF; structures in this room are "
+            "some of the few that cannot be destroyed by Power Bombs.\nNote that some game versions change that."
```

Comment:
`Note that some game versions change that.` What versions change this behavior and for which blocks? I cannot find them in [version differences.](https://tcrf.net/Metroid_Prime/Version_Differences)

Reply:
This was added at [Miepee's request](https://ptb.discord.com/channels/914291389293027329/1305369147881816076/1307433452114677823) - the version in question is the Archipelago version of Prime, if I understand correctly

Reply:
might be worth to say "different randomizers" instead